.github/pull_request_template.md

```diff
@@ -1,47 +1,43 @@
 Resolves #ISSUE(s).
-<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:
-
-Partially addresses:
-- ISSUE(s)
-
-Addresses sub-task (n) in:
-- ISSUE(s)
+<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address 
+the issue and you don't want it to auto-close, you can write one of the following instead of "Resolves"
+- Partially addresses
+- Addresses sub-task (n) in
 -->
 
-
-## Pre-merge checklist
-
-<!--- Docs: A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
-
-### PR Overview
-<!--- Short paragraph or short bulleted list describing changes in this PR -->
-
+## Overview
+<!--- A few sentences describing changes / what problem is solved. Should describe the expected result of the changes. 
+If applicable, point out the main change that solved the problem, e.g. fixed bug in method xyz. -->
 This PR:
 - One
 - Two
 - Three
 
-=====================
```

Comment:
## Divider
Not opposed to adding back, but I removed this because I thought it was visually redundant.

Reply:
yeah, no intent to have two ~horizontal rules